src/js/brandColors/brandColors.test.ts

```diff
@@ -7,403 +7,403 @@ describe('Brand Colors', () => {
   describe('White', () => {
     it('js tokens for white000 matches figma tokens white000', () => {
       expect(brandColors.white['000']).toStrictEqual(
-        designTokens.global.brandColors.white.white000.value.toLowerCase(),
+        designTokens.global.brandColors.white.white000.value,
```

Comment:
Now that the brand color hex values are all uppercase there is no need for this toLowerCase method